interview-questions.md

```diff
@@ -9,47 +9,65 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
 Your answer:
+JSX is a javascript markdown syntax extension that works similar like HTML. JSX is a building block of REACT components and what allows React to create user interfaces. It looks like a combination of JavaScript and an extended set of HTML.
 
 Researched answer:
+JSX is a React extension to Javascript. It allows JavaScript and HTML to co-exist. This allows component rendering using syntax similar to HTML.
 
 2. What is yarn? What file(s) are modified in a React application when you run yarn?
 
-Your answer:
+Your answer: 
+A react app is made of thousands of code snippets called node modules, that has to be organized in a certain way. So in order to create a new React app we have to download a package containing these modules. Yarn (yet another resource negotiator) is the command that will download these modules and create a file structure for a React app. 
 
 Researched answer:
+YARN is a package that contains node modules , it allows the use and share of code with developers around the world. The code is shared through a package and includes j.son file which describes the package. 
```

Comment:
There is also a file called yarn.lock I want you to be familiar with and would be a point of issue in an interview.